src/container/srv_target.c

```diff
@@ -323,21 +276,40 @@ cont_child_aggregate(struct ds_cont_child *cont, uint64_t *msecs)
 	if (epoch_max >= cont->sc_aggregation_max)
 		epoch_max = cont->sc_aggregation_max - 1;
 
+	if (param->ap_max_eph_get) {
+		uint64_t max_eph = param->ap_max_eph_get(cont);
+
+		epoch_max = min(epoch_max, max_eph);
+		if (epoch_min > epoch_max) {
```

Comment:
epoch_min >= epoch_max?

Reply:
Hmm, according to the original logic epoch_min == epoch_max will not cause aggregation to exit here? no?